client/me/purchases/remove-plan-dialog/get-plan-features.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+	isMonthly,
+	isWpComBusinessPlan,
+	isWpComEcommercePlan,
+	isWpComPersonalPlan,
+	isWpComPremiumPlan,
+} from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+
+export default function getPlanFeatures(
+	productSlug: string | undefined,
+	translate: ReturnType< typeof useTranslate >,
+	withinFirstYear: boolean,
+	hasDomain: boolean,
+	domainSlug: string
+): string[] {
+	if ( ! productSlug ) {
+		return [];
+	}
+
+	const isMonthlyPlan = isMonthly( productSlug );
+
+	/**
+	 * Define feature labels.
+	 */
+	const freeDomainFirstYear = String( translate( 'Your free domain for the first year' ) );
+	const freeDomain = String(
+		translate( 'Your free domain www.%(slug)s', {
+			args: { slug: domainSlug },
+		} )
+	);
+	const bestInClassHosting = String( translate( 'Best-in-class hosting' ) );
+	const addFreeSite = String( translate( 'An add-free site' ) );
```

Comment:
Should be "ad-free"

Reply:
this was fixed, the copy structure refatore, see 

> I'm not sure how far along this is, but I tried it and the main thing I noticed is that I couldn't trigger it in the way I expected to:
> 
> * Purchased a plan
> * Visited the purchase management page
> * Clicked "Cancel Subscription and Refund" to remove the plan
> * Didn't see the dialog anywhere during the process of removing the plan
> 
> In order to see it, I needed a plan that was no longer refundable and that already had auto-renew disabled (in other words, was already "cancelled" in some sense). Then I saw the dialog when I went to completely remove the subscription afterwards. Is that expected?
> 
> Also noticed that when I tried to remove a Pro plan, the feature list is missing:
> 
> ![pro-plan](https://user-images.githubusercontent.com/235183/188746166-c730a0a7-6a96-4db8-97ca-751fb83659f6.png)
> 
> I assume we should add a feature list for Pro and Starter (since there are plenty of those around that people might cancel), but either way, there should also be a fallback where if an unrecognized plan is being cancelled, the "If you cancel your plan, you will lose" sentence doesn't display.

The logic changed, the modal now support subscription and you should be able to access the modal consinstently.
PRO and Starte plans were added.